targets/ar71xx-mikrotik

```diff
@@ -1,9 +1,16 @@
 config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 
+ATH10K_PACKAGES='kmod-ath10k-ct ath10k-firmware-qca9887 ath10k-firmware-qca988x'
```

Comment:
I'm not sure about the firmware, that you're including here.
> Wireless 5 GHz chip model: QCA9892

Reply:
What should be the right firmware? This one works

Reply:
Interesting, I don't see a firmware matching the 9892 number, but if that one works, so be it.

Reply:
@mweinelt The 3rd digit indicates "SoHo / Enterprise". They re basically identical in terms of required driver.

Reply:
Okay, so drop  `ath10k-firmware-qca9887`.